compiler/Eta/Main/GhcMake.hs

```diff
@@ -139,7 +139,7 @@ depanal excluded_mods allow_dup_roots = do
 -- when building a library, so that GHC warns user about modules, not listed
 -- neither in `exposed-modules`, nor in `other-modules`.
 --
--- Here "home module" means a module, that doesn't come from an other package.
+-- Here "home module" means a module, that doesn't come from another package.
```

Comment:
This is my tooling flagging a suspect string. Note that I'm not performing any real grammar checking (and this PR is already medium sized, so I wouldn't want to here), but I did confirm the check appeared correct and applied this change.